modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/metadata/MetadataScopeAdapter.java

```diff
@@ -26,15 +24,19 @@ default boolean isCustomScope() {
     return hasOutputResolver() || hasInputResolvers();
   }
 
+  boolean isPartialKeyResolver();
```

Comment:
what if the is no key resolver? what does this method return?

Reply:
It should return false